templates/bookmarkingResume.jsx

```diff
@@ -8,6 +8,7 @@ export default function Bookmarking(props) {
   return (
     <div className="bookmarking-resume__inner">
       <button
+        type='button'
```

Comment:
Why?

Reply:
Decision was made after reading MDN doc. See Note 3: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#notes

Reply:
I see. It would only be an issue if injected into a form. Fair enough.